benchmarks/src/sha3-rocc.c

```diff
@@ -47,7 +49,6 @@ int main() {
     }
   } while(0);
 
-  
-  printf("success!\n");
+  printf("Success! Completed in %lu cycles\n", rdcycle());
```

Comment:
Using the bare value of the `cycle` CSR is not correct here, especially in a Linux environment where the benchmark is not the first program to execute after reset.  This should be a cycle difference representing only the cycles elapsed for the actual RoCC or scalar computation.  The cycle count should also be printed before the result check, since it's still interesting information eventhe  if test failed.

Reply:
Sure. I just thought this was easier. But that will make sure sense!